src/client/client.cpp

```diff
@@ -1774,13 +1774,25 @@ void Client::makeScreenshot()
 	char timetstamp_c[64];
 	strftime(timetstamp_c, sizeof(timetstamp_c), "%Y%m%d_%H%M%S", tm);
 
-	std::string filename_base = g_settings->get("screenshot_path")
+	std::string screenshot_dir;
+
+	if (fs::IsPathAbsolute(g_settings->get("screenshot_path"))) {
+		screenshot_dir = g_settings->get("screenshot_path");
+	} else {
+		screenshot_dir = porting::path_user + DIR_DELIM + g_settings->get("screenshot_path");
+	}
+
+	std::string filename_base = screenshot_dir
 			+ DIR_DELIM
```

Comment:
What happens if this becomes `C:\path\to\screenshots\\`, or `/path/to/screenshots//`?

Reply:
Is there a "plus_file" method of sorts that can be used to avoid issues with trailing slashes?

Reply:
At least Linux removes duplicated DIR_DELIMs automatically. I hope Windows does the same.

Reply:
Windows deletes trailing slashes and backslashes, but it does not delete multiple slashes in the middle, e.g. `path/to/dir//`,`path\to\dir\\`, and `path/to\dir/\` all work, but `path//to/dir` doesn't.